src/Tools/dotnet-monitor/Program.cs

```diff
@@ -59,6 +57,14 @@ class Program
                 Argument = new Argument<bool>(name: "metrics", defaultValue: true )
             };
 
+        private static Option TransportPath() =>
+            new Option(
+                alias: "--transport-path",
```

Comment:
I think `--server-address` or `--reverse-address` might be more identifiable from a user perspective.